pythonforandroid/bootstraps/service_library/build/templates/Service.tmpl.java

```diff
@@ -36,8 +36,27 @@ public static void prepare(Context ctx) {
     }
 
     public static void start(Context ctx, String pythonServiceArgument) {
-        Intent intent = getDefaultIntent(ctx, pythonServiceArgument);
-
+        Intent intent = getDefaultIntent(ctx,  "", "{{ args.name }}",
+                                         "{{ name|capitalize }}",
+                                         pythonServiceArgument);
+        //foreground: {{foreground}}
+        {% if foreground %}
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            ctx.startForegroundService(intent);
+        } else {
+            ctx.startService(intent);
+        }
+        {% else %}
+        ctx.startService(intent);
+        {% endif %}
```

Comment:
What about calling `start(ctx, "", "{{ args.name}}", "{{ name|capitalize }}", pythonServiceArgument)` instead of keeping duplicated code?